zipline/lib/adjust.pyd

```diff
@@ -0,0 +1,607 @@
+cdef class Adjustment:
```

Comment:
this file is totally named incorrectly, it shouldn't have been committed at all.